disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/exporters/NoopExporterCallback.java

```diff
@@ -8,7 +8,7 @@
 import io.opentelemetry.contrib.disk.buffering.SignalType;
 import javax.annotation.Nullable;
 
-final class NoopExporterCallback implements ExporterCallback {
+class NoopExporterCallback implements ExporterCallback {
```

Comment:
Keep `final` even if it's package private.

```suggestion
final class NoopExporterCallback implements ExporterCallback {
```